packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -324,6 +322,74 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
   }
 }
 
+/// Test
+class RenderSliverFixedVariableExtentList extends RenderSliverFixedExtentBoxAdaptor{
+  /// Test
+  RenderSliverFixedVariableExtentList({
+    @required RenderSliverBoxChildManager childManager,
+    List<double> itemExtents,
+  }) : super(childManager: childManager) {
+    this.itemExtents = itemExtents;
+  }
+
+  /// Test
+  List<double> get itemExtents => _itemExtents;
+  List<double> _itemExtents;
+  set itemExtents(List<double> value) {
+    assert(value != null);
+    if (_itemExtents == value)
```

Comment:
This will cause problems if somebody caches the itemExtent list in e.g. a stateful widget and then appends/removes items from it. Even though the list has changed, this would still evaluate to true.